File: Python/Iniciante/1020.py
import math

dias = int(input())

ano = 0
mes = 0
dia = 0

ano = math.floor(dias / 365)
dias %= 365

mes = math.floor(dias / 30)
dias %= 30

dia = dias

print(f"{ano} ano(s)\n{mes} mes(es)\n{dia} dia(s)")

File: JavaScript/Iniciante/1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

dia = lines.shift()
dia = parseInt(dia)

mes = 0
ano = 0

ano = Math.floor(dia / 365)
dia %= 365

mes = Math.floor(dia / 30)
dia %= 30

console.log(`${ano} ano(s)\n${mes} mes(es)\n${dia} dia(s)`)
